src/main/webapp/js/components/csv/csv-config-create.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import 'react-table/react-table.css';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import Select from '../../utils/inputs/select';
+import {
+  SITE_ENTITY,
+  VACCINEE_ENTITY,
+  VISIT_ENTITY,
+} from '../../constants/entity-types';
+import CsvConfigPage from './csv-config-page';
+import { createCsvConfig } from '../../actions/csv-config-actions';
+
+const ENTITY_TYPES = [
+  { label: 'Vaccinee', value: VACCINEE_ENTITY },
+  { label: 'Visit', value: VISIT_ENTITY },
+  { label: 'Site', value: SITE_ENTITY },
+];
+
+class CsvConfigCreate extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      entity: '',
+    };
+  }
+
+  onChange = (entity) => {
+    this.setState({ entity });
+  };
+
+  onSubmit = (values) => {
+    this.props.createCsvConfig(this.state.entity, values);
+  };
+
+  render() {
+    const csvConfig = {};
+    csvConfig.entity = this.state.entity;
```

Comment:
set it when creating the object 
`{ entity: this.state.entity }`
you don't need a separate variable for this, you can set it directly in the CsvConfigPage property